pkg/ddl/backfilling_dist_executor.go

```diff
@@ -251,14 +252,20 @@ func (*backfillDistExecutor) IsIdempotent(*proto.Subtask) bool {
 }
 
 func isRetryableError(err error) bool {
+	errMsg := err.Error()
+	for _, m := range dbterror.ReorgRetryableErrMsgs {
+		if strings.Contains(errMsg, m) {
+			return true
+		}
+	}
 	originErr := errors.Cause(err)
 	if tErr, ok := originErr.(*terror.Error); ok {
 		sqlErr := terror.ToSQLError(tErr)
 		_, ok := dbterror.ReorgRetryableErrCodes[sqlErr.Code]
 		return ok
 	}
-	// can't retry Unknown err.
-	return false
+	// For the unknown errors, we should retry.
+	return true
```

Comment:
for https://github.com/pingcap/tidb/pull/43782, only pick this part to make the test pass